apps/prairielearn/src/lib/db-types.ts

```diff
@@ -282,6 +282,30 @@ export const AssessmentSetSchema = z.object({
 });
 export type AssessmentSet = z.infer<typeof AssessmentSetSchema>;
 
+export const AuditEventSchema = z.object({
+  action: z.string(),
+  action_detail: z.string().nullable(),
+  agent_authn_user_id: IdSchema.nullable(),
+  agent_user_id: IdSchema.nullable(),
+  assessment_id: IdSchema.nullable(),
+  assessment_instance_id: IdSchema.nullable(),
+  assessment_question_id: IdSchema.nullable(),
+  context: z.record(z.string(), z.any()),
```

Comment:
👀 what is this column intended to be used for? Is this the equivalent of `data` in PT? If so, I'm wondering if we can use the same name for consistency.